### PR TITLE
fix nil speaker error

### LINKED ISSUE #167: Speaker in activities aren't implemented correctly
Currently, `speaker` allows `nil` values. Unfortunetelly, `to_markdown` and `to_plaintext` values can't handle `nil` values and cause `500: Internal Server Error`.

### PR DESCRIPTION
**Pull request by machadovilaca**
Couldn't reproduce the issue but I think this fixes #167

### CODE REVIEW

**Comment by marcelo140** [comment]
One more thing, currently the speaker feels like it's part of the activity's description (both in index and show). What do you think of highlighting it by putting it on info panel?

**Comment by machadovilaca** [comment]
@marcelo140 How does it look?
index
show

**Comment by marcelo140** [comment]
I'm ok with `show` but I still don't like it's positioning on the `index`. Can we just not show the speaker on the `index`? @martinhoaragao @dinispeixoto opinions on this ?

File: app/views/shared/activity/_description_date.slim [hunk 1]
```diff
@@ -1,9 +1,10 @@
-p.lead.activity-description
-  = to_plaintext(activity.speaker)
-
 p.lead.activity-description
   = truncate(to_plaintext(activity.description), length: 140)
 
+- if activity.speaker
+  i.far.fa-user
+    = " " + to_plaintext(activity.speaker)
```
Comment (ZePedroResende) [inline app/views/shared/activity/_description_date.slim]: this doesn't add an empty space ?

Comment (machadovilaca) [reply app/views/shared/activity/_description_date.slim]: that's the objective, anyway if the "look" is approved I will change it to an acceptable way to add space between the icon and the name

Comment (ZePedroResende) [reply app/views/shared/activity/_description_date.slim]: Ok that's what i was gonna sugest . good job

**Comment by ZePedroResende** [comment]
@machadovilaca  ping

**Comment by machadovilaca** [comment]
@ZePedroResende waiting for views approval

**Comment by ZePedroResende** [comment]
@machadovilaca  Approved

**Pull request merged**